mmpose/models/keypoint_heads/top_down_base_head.py

```diff
@@ -34,7 +36,20 @@ def forward(self, **kwargs):
     def inference_model(self, **kwargs):
         """Inference function."""
 
-    def decode_keypoints(self, img_metas, output_heatmap):
+    def decode_keypoints(self, img_metas, output_heatmap, img_size):
+        """Decode keypoints from heatmaps.
+
+        Args:
+            img_metas (list(dict)): Information about data augmentation
+                By default this includes:
+                - "image_file: path to the image file
+                - "center": center of the bbox
+                - "scale": scale of the bbox
+                - "rotation": rotation of the bbox
+                - "bbox_score": score of bbox
+            output_heatmap (np.ndarray[N, K, H, W]): model predicted heatmaps.
+            img_size (tuple(img_width, img_height)): model input image size.
```

Comment:
is `img_size` used?

Reply:
Not used for now. Just to keep the interface consistent.

Reply:
ok